commons-core/src/main/scala/com/avsystem/commons/serialization/json/JsonStringOutput.scala

```diff
@@ -1,18 +1,24 @@
 package com.avsystem.commons
 package serialization.json
 
-import com.avsystem.commons.serialization.{GenCodec, ListOutput, ObjectOutput, Output}
+import com.avsystem.commons.serialization.{GenCodec, IsoInstant, ListOutput, ObjectOutput, Output}
 
 object JsonStringOutput {
-  def write[T: GenCodec](value: T): String = {
+  def write[T: GenCodec](value: T, options: JsonOptions = JsonOptions.Default): String = {
     val sb = new JStringBuilder
-    GenCodec.write[T](new JsonStringOutput(sb), value)
+    GenCodec.write[T](new JsonStringOutput(sb, options), value)
     sb.toString
   }
 }
 
 trait BaseJsonOutput {
-  protected final def writeJsonString(builder: JStringBuilder, str: String): Unit = {
+  protected final def indent(builder: JStringBuilder, options: JsonOptions, depth: Int): Unit =
+    options.indentSize match {
```

Comment:
`.foreach`

Reply:
Again, I don't want that lambda in here - this is called often and I want this as fast as possible.